@here/harp-mapview/lib/MapView.ts

```diff
@@ -2517,6 +2528,10 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_elevationProvider;
     }
 
+    get shadowsEnabled(): boolean {
```

Comment:
Is there a reason to not have a setter?

Reply:
@ninok , I added a setter, but there is a bug in three.js, enabling and then disabling doesn't stop rendering the shadows, I will create an issue in three.js

Reply:
as discussed, I have added a call to `clearTileCache`